TicTacToe2/TicTacToe/Board.cs

```diff
@@ -6,5 +6,74 @@ namespace TicTacToe2.TicTacToe
 {
     class Board
     {
+        private char[,] space; //Board
+        private int gridNumber = 0; //Board
```

Comment:
A couple of minor naming recommendations:

* Names of array variables should generally be plural
* "size" would be a more standard name than "gridNumber" in this situation